apiDocs/wrhs-spec.md

```diff
@@ -175,22 +175,26 @@ Cancel the specified build.
 
 ### /builds/{pkg}
 
-#### GET
+#### PUT
 ##### Summary:
 
-Gets the builds for a package.
+Puts a fully built npm publish payload structure
 
 ##### Parameters
 
 | Name | Located in | Description | Required | Schema |
 | ---- | ---------- | ----------- | -------- | ---- |
 | pkg | path | The package name | Yes | string |
+| _attachments | requst body | The fully built asset | No | object |
+
+Note: The fully built asset can be both tarball and non-tarball content as long as
```

Comment:
Can you elaborate on what we'll do with a tarball here? Will we extract it? Are there any special files that could be in it? What exact tar format? gzip? Just plain tar?